teamB/kimSungHoon/app.js

```diff
@@ -0,0 +1,68 @@
+const dotenv = require('dotenv').config();
```

Comment:
- dotenv 의 경우 const 선언을 하지 않고 바로 호출과 적용이 가능합니다. 아래와 같이 수정해주세요 ~!
```suggestion
require('dotenv').config();
```